CHANGES.md

```diff
@@ -48,6 +49,9 @@ To be released.
  -  `Swarm<T>` became not to receive states from trusted peers.
     [[#1061], [#1102]]
  -  `Swarm<T>` became not to retry when block downloading.  [[#1062], [#1102]]
+ -  `Block<T>.Validate()` became to validate only `Block<T>.TxHash` instead of its all transactions.  [[#1116]]
```

Comment:
As `Block<T>.Validate()` is `internal`, we mention `Block<T>.Evaluate()` rather than `Validate()` here.  Also, we probably should update the docs here too?:

https://github.com/planetarium/libplanet/blob/ae6833181e6a58c78c4e081d366f96e1ec9bc08a/Libplanet/Blocks/Block.cs#L501-L526